src/modules/faqs/dto/createFaqsDto.ts

```diff
@@ -0,0 +1,17 @@
+import { IsOptional, IsString } from 'class-validator';
+
+export class CreateFaqDto {
```

Comment:
can we also add the `@IsNotEmpty()` class validator.

>@IsNotEmpty()	Checks if given value is not empty (!== '', !== null, !== undefined).

Reply:
Alright boss